pkg/cmd/push/cmd.go

```diff
@@ -58,17 +62,30 @@ func (opts *pushOptions) complete(ctx context.Context, args []string) error {
 	}
 	opts.configHome = configHome
 
-	modelRef, extraTags, err := util.ParseReference(args[0])
+	srcRef, extraTags, err := util.ParseReference(args[0])
 	if err != nil {
-		return fmt.Errorf("failed to parse reference: %w", err)
-	}
-	if modelRef.Registry == "localhost" {
-		return fmt.Errorf("registry is required when pushing")
+		return fmt.Errorf("failed to parse reference %s: %w", args[0], err)
 	}
 	if len(extraTags) > 0 {
 		return fmt.Errorf("reference cannot include multiple tags")
 	}
-	opts.modelRef = modelRef
+	opts.srcModelRef = srcRef
+	if len(args) == 1 {
+		opts.destModelRef = srcRef
+	} else {
+		destRef, extraTags, err := util.ParseReference(args[1])
+		if err != nil {
+			return fmt.Errorf("failed to parse target reference %s: %w", args[1], err)
+		}
+		if len(extraTags) > 0 {
+			return fmt.Errorf("target reference cannot include multiple tags")
+		}
+		opts.destModelRef = destRef
+	}
+
+	if opts.destModelRef.Registry == "localhost" {
```

Comment:
`localhost` is a valid use case in local testing. Is this making it impossible?

Reply:
Nope :)

This check has always been [here](https://github.com/jozu-ai/kitops/blob/60e30a5b2423d3e16c4c99edf527438d6ca79109/pkg/cmd/push/cmd.go#L65-L67) -- it's because we adopted the podman convention of using `localhost` as the registry placeholder in references (so that they're still valid). Local usage isn't impacted, since you would likely tag your ModelKit `localhost:5050`, which doesn't match.

For context, we need a way to store and parse references to ModelKits that don't have full information, while the OCI spec requires both a `registry` and `repository`. For example, completely untagged ModelKits are represented `localhost/_@<digest>`.